sysconfigs/daint/packages.yaml

```diff
@@ -1,243 +1,400 @@
+      - cray-libsci_acc/20.06.1
+    - spec: cray-libsci_acc@20.06.1%cce
+      modules:
+      - cray-libsci_acc/20.06.1
+    - spec: cray-libsci_acc@20.06.1%pgi
+      modules:
+      - cray-libsci_acc/20.06.1
+  cuda:
+    version: ['10.2']
+    externals:
+    - spec: cuda@10.2%gcc
+      modules:
+      - cudatoolkit/10.2.89_3.28-2.1__g52c0314
+  curl:
+    externals:
+    - spec: curl@7.60.0
+      prefix: /usr
+  diffutils:
+    externals:
+    - spec: diffutils@3.6
+      prefix: /usr
+  elpa:
+    variants: +openmp +optflags
+  gawk:
+    externals:
+    - spec: gawk@4.2.1
+      prefix: /usr
+  gettext:
+    externals:
+    - spec: gettext@0.19.8.1
+      prefix: /usr
+  hdf5:
+    buildable: false
+    externals:
+    - spec: hdf5@1.12.0.0%cce~mpi+hl
+      prefix: /opt/cray/pe/hdf5/1.12.0.0/CRAYCLANG/9.0
+    - spec: hdf5@1.12.0.0%cce~mpi+hl+fortran
+      prefix: /opt/cray/pe/hdf5/1.12.0.0/CRAYCLANG/9.0
+    - spec: hdf5@1.12.0.0%intel~mpi+hl
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.0%gcc~mpi+hl
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.0%cce~mpi+hl
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.0%pgi~mpi+hl
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.0%intel+mpi+hl
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.0%gcc+mpi+hl
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.0%cce+mpi+hl
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.0%pgi+mpi+hl
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.0%intel~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.0%gcc~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.0%cce~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.0%pgi~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.0
+    - spec: hdf5@1.12.0.4%intel~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.4
+    - spec: hdf5@1.12.0.4%gcc~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.4
+    - spec: hdf5@1.12.0.4%cce~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.4
+    - spec: hdf5@1.12.0.4%pgi~mpi+hl+fortran
+      modules:
+      - cray-hdf5/1.12.0.4
+    - spec: hdf5@1.12.0.0%intel+mpi+hl+fortran
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.0%gcc+mpi+hl+fortran
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.0%cce+mpi+hl+fortran
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.0%pgi+mpi+hl+fortran
+      modules:
+      - cray-hdf5-parallel/1.12.0.0
+    - spec: hdf5@1.12.0.4%intel+mpi+hl+fortran
+      modules:
+      - cray-hdf5-parallel/1.12.0.4
+    - spec: hdf5@1.12.0.4%gcc+mpi+hl+fortran
+      modules:
+      - cray-hdf5-parallel/1.12.0.4
+    - spec: hdf5@1.12.0.4%cce+mpi+hl+fortran
```

Comment:
Question:
Why do we need the new entries: spec and prefix

Is this required by the new Spack-version?

Reply:
It's required by the new version. I did nothing new apart reformating using the function the give us ;)